src/components/BackgroundPlaceholder/BackgroundPlaceholder.tsx

```diff
@@ -32,13 +33,13 @@ const EditBgIcon = styled.img`
 
 type BackgroundPlaceholderProps = {
   alt: string;
-  background?: string;
+  background?: string | ImageRef | null;
```

Comment:
Hmm, I thought we device to separate backgroundUpload (ImageRef) from the background itself (string). When You fetching the BE You got string for that field, but when You are editing then You should update to fields backgroundUpload with ImageRef and in the same time background field which will have a value backgroundUpload.blobUrl.